internal/hcsoci/containerex.go

```diff
@@ -43,11 +42,12 @@ const (
 type CreateOptions struct {
 
 	// Common parameters
-	ID            string                       // Identifier for the container
-	Owner         string                       // Specifies the owner. Defaults to executable name.
-	Spec          *specs.Spec                  // Definition of the container or utility VM being created
-	SchemaVersion *schemaversion.SchemaVersion // Requested Schema Version. Defaults to v2 for RS5, v1 for RS1..RS4
-	HostingSystem *uvm.UtilityVM               // Utility or service VM in which the container is to be created.
+	ID               string                       // Identifier for the container
+	Owner            string                       // Specifies the owner. Defaults to executable name.
+	Spec             *specs.Spec                  // Definition of the container or utility VM being created
+	SchemaVersion    *schemaversion.SchemaVersion // Requested Schema Version. Defaults to v2 for RS5, v1 for RS1..RS4
+	HostingSystem    *uvm.UtilityVM               // Utility or service VM in which the container is to be created.
+	NetworkNamespace string                       // Host network namespace to use (overrides anything in the spec)
```

Comment:
I don't think there's any choice for now, but is this the right thing? Perhaps we could use an annotation in the spec, or have an endpoint list item in the spec encoded as `namespace:id` to distinguish it? Can always change later though.